### PR TITLE
Explain a bit more about Prioritization summaries

### PR DESCRIPTION
**Pull request by spastorino**
r? @XAMPPRocky but wait for someone else from @rust-lang/wg-prioritization to approve in particular @apiraino

### CODE REVIEW

File: src/compiler/prioritization/procedure.md [hunk 1]
```diff
@@ -131,6 +131,14 @@ We will have checkins from *WG-X* and *WG-Y*
 
 ### Add details to the Agenda
 
+In order to summaries issues take into account that is important to add
```
Comment (o0Ignition0o) [inline src/compiler/prioritization/procedure.md:134]:
```suggestion
In order to summarize issues take into account that it is important to add
```

**Review by Stupremee — COMMENTED** [review]
We could probably add an example for a "perfect" summary or add a few basic facts that are important to add (like regressed PR / commit, Author, etc)

Edit: If we add this, we should use a real issue to showcase a summary

File: src/compiler/prioritization/procedure.md [hunk 2]
```diff
@@ -131,6 +131,14 @@ We will have checkins from *WG-X* and *WG-Y*
...
+details that make the meeting be more effective and faster. For that
```
Comment (o0Ignition0o) [inline src/compiler/prioritization/procedure.md:135]:
```suggestion
details that will speed up the meeting. For that
```

File: src/compiler/prioritization/procedure.md [hunk 3]
```diff
@@ -131,6 +131,14 @@ We will have checkins from *WG-X* and *WG-Y*
...
+consider mentioning important actors, important details like is a
```
Comment (o0Ignition0o) [inline src/compiler/prioritization/procedure.md:136]:
```suggestion
consider mentioning important actors, important details like if it is a
```

File: src/compiler/prioritization/procedure.md [hunk 4]
```diff
@@ -131,6 +131,14 @@ We will have checkins from *WG-X* and *WG-Y*
...
+regression and when it has regressed or if it already has a PR.
```
Comment (o0Ignition0o) [inline src/compiler/prioritization/procedure.md:137]:
```suggestion
regression and when it has regressed or if it already has a PR attached to it.
```

**Review by o0Ignition0o — APPROVED** [review]

**Review by o0Ignition0o — COMMENTED** [review]
missclicked the approve button

**Comment by JamesPatrickGill** [comment]
> We could probably add an example for a "perfect" summary or add a few basic facts that are important to add (like regressed PR / commit, Author, etc)

I agree, example would be super useful for a newcomer to see the reason for the existence of these summaries.

File: src/compiler/prioritization/procedure.md [hunk 5]
```diff
@@ -131,6 +131,14 @@ We will have checkins from *WG-X* and *WG-Y*
...
+Properly read the issue to summarize it and also process the
+Prioritization WG stream topic related to the issue to extract other
+useful information.
```
Comment (LeSeulArtichaut) [inline src/compiler/prioritization/procedure.md:140]: We also summarize beta/stable nominations which have PRs and as such don't get discussed. Not sure how to make it more clear in the text here though

**Review by LeSeulArtichaut — APPROVED** [review]
I think this section helps understanding why we do summaries, so 👍
I personally find the wording a bit unclear sometimes but that could be improved in a separate PR.

**Comment by apiraino** [comment]
So, if I understand the goal here is to provide an additional brief digest about the issue, in addition to simple "facts" retrieved from the issue (such as reporter, assignee, linked PRs, should solve this, etc.) as detailed [here](https://forge.rust-lang.org/compiler/prioritization/procedure.html#summarize-stablebeta-nominations), which we already do to the best of our knowledge.

I'm all for helping the team acquiring a context about the issue, but I'm afraid that adding more prose, such as a "digest" meant to be really meaningful, could end up having an agenda with wall of texts which (imo) would be opposite of the indent here.

Second point, digesting an issue is **hard**. Sometimes the context is stratified, spread on different issues, PRs and comments are from skilled people who are deep into the problem. I, for one, find hard to get the context of the Rust compiler issues, code samples provided are often opaque to me so I rely on comments and linked issues to figure a bit of what is all about and often that's as far as I can go.

I guess what I'm trying to say is that it's fine to try adding more info, but _too much_ info would end up being noisy.

**Comment by spastorino** [comment]
> We could probably add an example for a "perfect" summary or add a few basic facts that are important to add (like regressed PR / commit, Author, etc)

Agree about the example, maybe we should merge this and if someone can work on an example about it it would be great.

**Comment by spastorino** [comment]
> I think this section helps understanding why we do summaries, so +1
> I personally find the wording a bit unclear sometimes but that could be improved in a separate PR.

Yeah, please do fix my wording here :heart:

File: src/compiler/prioritization/procedure.md (hunk 5 above)
Comment (spastorino) [reply src/compiler/prioritization/procedure.md:140]: Yeah agree, maybe writing in a more general way would be better. I guess we should merge this and keep improving the text in separate PRs.

**Comment by spastorino** [comment]
> I'm all for helping the team acquiring a context about the issue, but I'm afraid that adding more prose, such as a "digest" meant to be really meaningful, could end up having an agenda with wall of texts which (imo) would be opposite of the indent here.

Agree with this, we can also close the PR and open a new one approaching a different strategy. But :+1: that the idea with the summaries is to help and having something short, that gives hints that go right to the point is the most desirable thing to do, rather than have walls of texts around.

**Comment by Stupremee** [comment]
> So, if I understand the goal here is to provide an additional brief digest about the issue, in addition to simple "facts" retrieved from the issue (such as reporter, assignee, linked PRs, should solve this, etc.) as detailed [here](https://forge.rust-lang.org/compiler/prioritization/procedure.html#summarize-stablebeta-nominations), which we already do to the best of our knowledge.

We can just add some real (or fake) data to it, and then we can use it as a standard example for a summary.

**Comment by apiraino** [comment]
@spastorino based on the feedback received so far (see [Zulip topic](https://zulip-archive.rust-lang.org/238009tcompilermeetings/45307Suggestionstoimprovetheweeklymeetingagenda.html)) I think we can close this proposal.

**Pull request closed without merging**